include/llvm/Analysis/TargetLibraryInfo.def

```diff
@@ -431,9 +431,6 @@ TLI_DEFINE_STRING_INTERNAL("fmodl")
 /// FILE *fopen(const char *filename, const char *mode);
 TLI_DEFINE_ENUM_INTERNAL(fopen)
 TLI_DEFINE_STRING_INTERNAL("fopen")
-/// FILE *fopen64(const char *filename, const char *opentype)
```

Comment:
Maybe we should wrap them with `#ifdef 0 // HLSL Change` instead of deleting them completely. For all the following files.

Reply:
hmmm... I should have thought of this too. I agree.